Sources/MongoSwift/BSON/BsonValue.swift

```diff
@@ -277,51 +231,56 @@ extension Date: BsonValue {
         }
     }
 
-    public static func from(iter: inout bson_iter_t) -> BsonValue {
-        return Date(msSinceEpoch: bson_iter_date_time(&iter))
+    public init(from iter: DocumentIterator) throws {
+        self.init(msSinceEpoch: bson_iter_date_time(&iter.iter))
     }
 }
 
 /// An internal struct to represent the deprecated DBPointer type. While DBPointers cannot
 /// be created, we may need to parse them into `Document`s, and this provides a place for that logic.
 internal struct DBPointer: BsonValue {
 
-    public var bsonType: BsonType { return .dbPointer }
+    var bsonType: BsonType { return .dbPointer }
 
-    public func encode(to storage: DocumentStorage, forKey key: String) throws {
-        throw MongoError.bsonEncodeError(message: "DBPointers are deprecated; use a DBRef instead")
+    func encode(to storage: DocumentStorage, forKey key: String) throws {
```

Comment:
By removing the visibility here and on `var bsonType` above, do both of these now default to `internal` as well? If so, how would they be called?

Likewise a question for Symbol below.

Reply:
Ah, an `internal` class could never be used outside of the module itself, so I suppose it makes sense that its methods could never use a more public visibility (although you could certainly get more restrictive like `private` or `fileprivate`).

Reply:
yup 👍